### DISCUSSION TITLE
Equipo 17 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#25] [Proyecto Final]
Hola Equipo 17, @AmbarPranahuti @Antuuu08 

Aquí está el hilo para coordinar entre ustedes, charlar con el Equipo de Educación, recibir retroalimentación sobre cada etapa.

Aquí está su carpeta grupal en nuestro repositorio compartido: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/tree/main/equipo-17
La carpeta tiene formularios de entrega para cada etapa.

Con la emoción de seguir el desarrollo de su proyecto,
Kasturi

**Comment by AmbarPranahuti** [comment]
Holaaaaa Al final no entendí, somos tres en el proyecto, ¿no?
Por cierto, cuantas etapas ya tienes? Yo tengo la cero y la uno porque crei que no tenía equipo

**Reply by KasturiSShah** [reply]
Hola @AmbarPranahuti @Antuuu08 buenos días
Perdón la demora en contestar. Para aclarar, parece que el tercer miembro del equipo prefiere hacer el proyecto sola, entonces ahora ustedes dos forman el equipo 17. 
Si ambas tienen las etapas 0 y 1 completadas, me avisan si prefieren juntar las etapas y avanzar con el proyecto en grupo, o si prefieren hacer el proyecto solas. Si quisieran hacer el protecto como grupo, ¿podrían entregar las etapas 0 (y 1, si fuera posible) tan pronto como sea posible? Aquí está la carpeta en Github donde necesitan entregarla: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/tree/main/equipo-17
Mucha suerte 😄 
Kasturi
P.D. Estoy dando una prórroga a ustedes hasta este domingo para la etapa 2, ¿está bien así?

**Comment by Antuuu08** [comment]
Yo también tengo ambas fases  , les parece si se unen a este grupo para
tener mejor coordinación  https://chat.whatsapp.com/GRkDAXY24rFIhU9Ykd1foM.

**Comment by CtrlRachie** [comment]
¡Hola equipo 17! 👋🏼💜

Espero que estén bien. Paso por aquí para saber cómo van con el desarrollo del proyecto, vi que solo han presentado el primer entregable en Github y estamos en las últimas semanas del curso. ¿Han enfrentado alguna complicación o desafío en el proceso? Recuerden que estoy aquí para brindarles cualquier ayuda que necesiten. 💪🏽
Cabe recalcar que de momento vi su video y su progreso y van por un muy buen camino 😊 

Sé que a veces los proyectos pueden ser retadores, pero no se preocupen, ¡juntos podemos superar cualquier obstáculo! 💪🏼 Si necesitan ayuda adicional, no duden en responder a este hilo o contactarme directamente. Estoy aquí para apoyarlas en lo que necesiten. 👍🏻

_"La clave está en no rendirse nunca, aunque todo parezca difícil."_ 💫

¡Vamos, equipo 17! Estoy seguro de que harán un excelente trabajo en las últimas etapas del proyecto. ¡A por la meta juntos! 🎯

Saludos cordiales,
Raquel Calderon
_Equipo de Patrones Hermosos_

**Comment by KasturiSShah** [comment]
Hola @AmbarPranahuti @Antuuu08 

Espero estén muy bien.

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día. Horario disponible [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing))

**_No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo._**

(Una recomendación: La etapa 5 no necesita que la etapa 3 ni 4 sea completada. Entonces, pueden pasar 30 minutos (es corta) completando la etapa 5, entregarla, y luego avanzar con las etapas 3 y 4.)

Cualquier pregunta sobre el proyecto final, déjanos saber. Quedamos atentas a tus actualizaciones para poder revisar las etapas / apoyarte en lo que podamos.

Mucha suerte, mucho éxito, hasta la clausura del curso,
Kasturi
